### PR TITLE
docs: highlight Git Credential Manager as alternative to PATs/SSH keys

### LINKED ISSUE #17200: Highlight Git Credential Manager as alternative to PATs/SSH keys
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

- [Using Two-Factor Authentication with the Command Line](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication#using-two-factor-authentication-with-the-command-line)
- [Creating a Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
- [About Remote Repositories](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls)

### What part(s) of the article would you like to see updated?

For [Using Two-Factor Authentication with the Command Line](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication#using-two-factor-authentication-with-the-command-line), I will add a new section about using [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) for 2FA instead of a PAT or SSH Key.

For [Creating a Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token), I will add a note to the top of the page calling out Git Credential Manager as an alternative to using personal access tokens.

For [About Remote Repositories](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls), I will clarify that, while using PATs is supported, a great alternative is using a credential manager, such as Git Credential Manager.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ldennington**
Git Credential Manager is a secure, cross-platform credential helper that
provides a user-friendly alternative to PATs and SSH keys. Updating the
docs to increase user awareness of this option.

### Why:

Closes  #17200

### What's being changed:

In [Using Two-Factor Authentication with the Command Line](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication#using-two-factor-authentication-with-the-command-line), I am adding add a new section about using [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) for 2FA instead of a PAT or SSH Key.

In [Creating a Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token), I am adding a note to the top of the page calling out Git Credential Manager as an alternative to using personal access tokens.

In [About Remote Repositories](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls), I clarify that, while using PATs is supported, a great alternative is using a credential manager, such as Git Credential Manager.

### Check off the following:

- [X] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@ldennington 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by mumoojung** [comment]
> Git Credential Manager is a secure, cross-platform credential helper that provides a user-friendly alternative to PATs and SSH keys. Updating the docs to increase user awareness of this option.
> 
> ### Why:
> Closes #17200
> 
> ### What's being changed:
> In [Using Two-Factor Authentication with the Command Line](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication#using-two-factor-authentication-with-the-command-line), I am adding add a new section about using [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) for 2FA instead of a PAT or SSH Key.
> 
> In [Creating a Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token), I am adding a note to the top of the page calling out Git Credential Manager as an alternative to using personal access tokens.
> 
> In [About Remote Repositories](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls), I clarify that, while using PATs is supported, a great alternative is using a credential manager, such as Git Credential Manager.
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
> * [x]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> * [ ]  This pull request impacts the contribution experience
>   
>   * [ ]  I have added the 'writer impact' label
>   * [ ]  I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

BingWallpaper

File: data/reusables/user-settings/password-authentication-deprecation.md [hunk 1]
```diff
@@ -1 +1 @@
-When Git prompts you for your password, enter your personal access token (PAT) instead.{% ifversion not ghae %} Password-based authentication for Git has been removed, and using a PAT is more secure.{% endif %} For more information, see "[Creating a personal access token](/github/authenticating-to-github/creating-a-personal-access-token)."
+Password-based authentication for Git will no longer work, as it has been removed in favor of more secure authentication methods. Instead use a credential helper like [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/f5bd57908b17e7772dde33d84c3a2634fb9d4768/README.md#git-credential-manager) or a [personal access token](/github/authenticating-to-github/creating-a-personal-access-token)."
```
Comment (mjcheetham) [inline data/reusables/user-settings/password-authentication-deprecation.md:1]: What about for GHAE? The original text had "if not GHAE" conditionals?

Also drop the "," from ".. no longer work, as it has .."

File: content/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication.md [hunk 2]
```diff
@@ -59,7 +59,11 @@ If you have installed and signed in to {% data variables.product.prodname_mobile
 
 ## Using two-factor authentication with the command line
 
-After you've enabled 2FA, you must use a personal access token or SSH key instead of your password when accessing {% data variables.product.product_name %} on the command line.
+After you've enabled 2FA, you will no longer use your password to access {% data variables.product.product_name %} on the command line. You can instead use [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md), a personal access token, or an SSH key.
```
[the remark this answers is not in the document]
Comment (ldennington) [reply content/authentication/securing-your-account-with-two-factor-authentication-2fa/accessing-github-using-two-factor-authentication.md:62]: I'll go ahead and update to that. Thanks!

File: data/reusables/user-settings/password-authentication-deprecation.md (hunk 1 above)
Comment (ldennington) [reply data/reusables/user-settings/password-authentication-deprecation.md]: Well-spotted, thank you.

**Comment by ramyaparimi** [comment]
@ldennington Sure! Thanks so much for the ping. I will get in touch with the team on this. Thanks so much for your patience 💛.

**Review by lecoursen — DISMISSED** [review]
Thanks @ldennington, and sorry for the delay! I'm going to apply a few suggestions, and I left one comment you'll have to address since I don't have access to your fork. 😊

**Comment by ldennington** [comment]
@lecoursen Thanks for the review! I went ahead and applied your suggested change, so hopefully we're good to go!

File: content/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token.md [hunk 3]
```diff
@@ -19,6 +19,11 @@ topics:
   - Access management
 shortTitle: Create a PAT
 ---
+{% note %}
+
+**Note:** [Git Credential Manager](https://aka.ms/gcm) is a secure, cross-platform alternative to using personal access tokens. Git Credential Manager is [simple to set up](https://aka.ms/gcm/installdocs) and eliminates the need to manage personal access token scope and expiration.
```
Comment (lecoursen) [inline content/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token.md:24]:
```suggestion
**Note:** [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) is a secure, cross-platform alternative to using personal access tokens (PATs), which eliminates the need to manage PAT scope and expiration. For installation instructions, see [Download and install](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md#download-and-install) in the GitCredentialManager/git-credential-manager repository.
```

File: data/reusables/user-settings/password-authentication-deprecation.md [hunk 4]
```diff
@@ -1 +1 @@
-When Git prompts you for your password, enter your personal access token (PAT) instead.{% ifversion not ghae %} Password-based authentication for Git has been removed, and using a PAT is more secure.{% endif %} For more information, see "[Creating a personal access token](/github/authenticating-to-github/creating-a-personal-access-token)."
+When Git prompts you for your password, enter your personal access token (PAT). Alternatively, you can use a credential helper like [Git Credential Manager](https://aka.ms/gcm).{% ifversion not ghae %} Password-based authentication for Git has been removed in favor of more secure authentication methods.{% endif %} For more information, see "[Creating a personal access token](/github/authenticating-to-github/creating-a-personal-access-token)."
```
Comment (lecoursen) [inline data/reusables/user-settings/password-authentication-deprecation.md:1]:
```suggestion
When Git prompts you for your password, enter your personal access token (PAT). Alternatively, you can use a credential helper like [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md).{% ifversion not ghae %} Password-based authentication for Git has been removed in favor of more secure authentication methods.{% endif %} For more information, see "[Creating a personal access token](/github/authenticating-to-github/creating-a-personal-access-token)." 
```

File: content/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token.md [hunk 5]
```diff
@@ -19,6 +19,11 @@ topics:
   - Access management
 shortTitle: Create a PAT
 ---
+{% note %}
+
+**Note:** [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) is a secure, cross-platform alternative to using personal access tokens (PATs), which eliminates the need to manage PAT scope and expiration. For installation instructions, see [Download and install](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md#download-and-install) in the GitCredentialManager/git-credential-manager repository.
```
Comment (lecoursen) [inline content/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token.md:24]: So sorry for the back and forth, but I didn't realize there's already a note right below this new one! Can you please combine then into one, like this?

**Notes**:

   - If you use {% data variables.product.prodname_cli %} to authenticate to {% data variables.product.product_name %} on the command line, you can skip generating a personal access token and authenticate via the web browser instead. For more information about authenticating with {% data variables.product.prodname_cli %}, see [`gh auth login`](https://cli.github.com/manual/gh_auth_login).
   - [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) is a secure, cross-platform alternative to using personal access tokens (PATs), which eliminates the need to manage PAT scope and expiration. For installation instructions, see [Download and install](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md#download-and-install) in the GitCredentialManager/git-credential-manager repository.

Thanks!

Comment (ldennington) [reply content/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token.md]: No problem. I think I've got it fixed up. Just let me know if there's anything else you notice!

File: data/reusables/user-settings/password-authentication-deprecation.md (hunk 4 above)
Comment (lecoursen) [inline data/reusables/user-settings/password-authentication-deprecation.md:1]:
```suggestion
When Git prompts you for your password, enter your personal access token (PAT). Alternatively, you can use a credential helper like [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md).{% ifversion not ghae %} Password-based authentication for Git has been removed in favor of more secure authentication methods.{% endif %} For more information, see "[Creating a personal access token](/github/authenticating-to-github/creating-a-personal-access-token)." 
```

**Review by lecoursen — APPROVED** [review]
Thanks for bearing with me on this one! It's looking great, so I'll merge now. ⚡

**Pull request merged**